src/views/drivingLicence/check-your-details.html

```diff
@@ -123,8 +123,6 @@ <h2>{{ translate("pages.check-your-details.correctCheck") }}</h2>
             id: "confirmDetails"
         }) }}
 
-        <p class="govuk-hint govuk-!-margin-top-0 govuk-!-padding-top-0"> {{ translate("fields.confirmDetails.items.detailsNotConfirmed.hint") }} </p>
```

Comment:
How come this has been removed?

Reply:
This was hint text that was appearing Below the radio button (but was seperate), this was because our radio buttons hints where not working (bug from hmpo-components which has now been resolved in a later version).

This ticket was raised to remove the hint text that we had originally used as a quick fix for the bug from hmpo-components. But now is irrelevant as the hint text linked to the radio button is working (covered by translations in yaml message files)